recipes-kernel/kernel-tests/kernel-performance-tests.bb

```diff
@@ -27,6 +27,16 @@ SRC_URI += "\
     file://test_kernel_cyclictest_hackbench.sh \
     file://test_kernel_cyclictest_fio.sh \
     file://test_kernel_cyclictest_iperf.sh \
+    file://test_kernel_cyclictest_idle_containerized.sh \
```

Comment:
Given this tests take a while to run (8 hours per test x 4 tests = 32 hours) can you split the containerized versions into a separate recipe? Maybe "kernel-performance-tests-containerized"  or something like that.
It will require separate ATS event/run but it will give us the option to run it in parallel (contingent on ATS HW availability etc.).